### PR TITLE
ELB: add policy for adding listener certificates

### PR DESCRIPTION
**Pull request by mandar242**
Added missing policies to add certificate to listener.

PR needing permissions: https://github.com/ansible-collections/amazon.aws/pull/1950

API reference [AddListenerCertificates](https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_AddListenerCertificates.html)

Failing task: https://github.com/ansible-collections/amazon.aws/pull/1950/files#diff-401031aa90b02aecfcae9b6abf5e0ad28d5adb08698bee29b161c3c4c6e014f3R55-R72

### CODE REVIEW

File: aws/policy/compute.yaml [hunk 1]
```diff
@@ -155,6 +155,7 @@ Statement:
       - autoscaling:TerminateInstanceInAutoScalingGroup
       - ec2:DeleteVolume
       - elasticloadbalancing:AddTags
+      - elasticloadbalancing:AddListenerCertificates
       - elasticloadbalancing:ApplySecurityGroupsToLoadBalancer
       - elasticloadbalancing:AttachLoadBalancerToSubnets
       - elasticloadbalancing:RemoveTags
```
Comment (alinabuzachis) [inline aws/policy/compute.yaml:161]: It has nothing to do with your change, but can you also please move this line `- elasticloadbalancing:RemoveTags` before `- elasticloadbalancing:RegisterTargets` to keep the alphabetical ordering?

File: aws/policy/compute.yaml [hunk 2]
```diff
@@ -155,6 +155,7 @@ Statement:
       - autoscaling:TerminateInstanceInAutoScalingGroup
       - ec2:DeleteVolume
       - elasticloadbalancing:AddTags
+      - elasticloadbalancing:AddListenerCertificates
```
Comment (alinabuzachis) [inline aws/policy/compute.yaml:158]: To keep the alphabetical ordering:
```suggestion
      - elasticloadbalancing:AddListenerCertificates
      - elasticloadbalancing:AddTags
```

**Review by alinabuzachis — APPROVED** [review]

**Comment by gravesm** [comment]
The linked PR is still failing for me with these changes. I think you need to add the `arn:aws:elasticloadbalancing:{{ aws_region }}:{{ aws_account_id }}:listener/*` resource to the rule.

**Comment by mandar242** [comment]
> The linked PR is still failing for me with these changes. I think you need to add the `arn:aws:elasticloadbalancing:{{ aws_region }}:{{ aws_account_id }}:listener/*` resource to the rule.

Added.

**Review by gravesm — APPROVED** [review]
Working now!

**Pull request merged**